snap/snapcraft.yaml

```diff
@@ -49,6 +65,16 @@ parts:
     stage-packages: [libzmq5]
     override-build: |
       go mod tidy
+
+      GIT_VERSION=$(git describe --tags --abbrev=0 | sed 's/v//')
+      if [ -z "$GIT_VERSION" ]; then
+        GIT_VERSION="0.0.0"
+      fi
+
```

Comment:
There are some redundant code for extracting the version.

I suggest the following:
1. Add `snapcraftctl set-version ${GIT_VERSION}` to line 73.
2. Remove the entire `version` part and adopt the version from `web-static` instead.